src/kayenta/stages/kayentaStage/kayentaStageExecutionDetails.html

```diff
@@ -11,6 +11,15 @@
       </div>
     </div>
 
+    <div class="row" ng-if="kayentaStageDetailsCtrl.canaryRuns.length < 50">
```

Comment:
Having trouble remembering all this, but I thought a `canaryRun` here is a canary judgment - I don't think recommending more than 50 is right here. I'm not totally sure that the number of judgments correlates with the quality of the canary.

Reply:
ah ok, in that case I don't know what the data points are that reflect accuracy (mentioned in this comment: https://github.com/spinnaker/kayenta/issues/283#issuecomment-380337009).  I've backed out this template addition.  Can add it when I know more.

Reply:
In the json source backing the stage execution view, there is a `result` object. It contains (within `judgeResult`) an array of `results` objects. Each contains these values: `controlMetadata.stats.count` and `experimentMetadata.stats.count`. At a glance, I think those are the values we would want to examine (note that the control and experiment counts should match or the comparison would have been rejected by kayenta anyway; so only one in each of the `results` objects needs to be considered).